src/controllers/users.js

```diff
@@ -12,6 +15,70 @@ class UsersController {
 
     return res.json(newUser);
   }
+
+  async deleteUser (req, res) {
+    try {
+      await User.deleteOne({ _id: req.params.id }).exec();
+
+      return res.status(200).json({ message: 'User is deleted.' });
+    } catch (error) {
+      res.status(400).json({ err: error });
+    }
+  }
+
+  async getUsers (req, res) {
+    const { _id, name, email, status, role, page, docs } = req.body;
+    const query = {};
+    if (_id != null) query._id = _id;
+    if (name != null) query.name = name;
+    if (email != null) query.email = email;
+    if (status != null) query.status = status;
+    if (role != null) query.role = role;
+    const pageNumber = page || PAGE_NUMBER;
+    const docsNumber = docs || DOCS_NUMBER;
+
+    try {
+      const users = await User.find(query)
+        .skip((pageNumber - 1) * docsNumber)
```

Comment:
All these counts must be performed on client side. In our case server must accept skip/take